Complete transition to using color_field for selecting colors

atomaka opened the pull request:
The labels color field was updated to use the `color_field` form helper in commit 7606b93.  This PR updates the broadcast messages form to do the same.

atomaka committed:
Update broadcast messages to use color_field

Razer6 commented:
@atomaka Thanks for this PR. Is the hint ` 6 character hex values starting with a # sign.` still needed now?

atomaka commented:
@Razer6 The other commit did not remove the hint so I left it for consistency.  I'm not sure if there is still a purpose.

Razer6 commented:
You can remove them. In both cases for consistency.

atomaka committed:
Update help texts and default value setting

atomaka commented:
I've removed the help texts from the broadcast messages forms and updated it in the labels creation form.  I also changed from setting the placeholder attribute on the form element to setting the value.  Placeholder is not relevant with color type fields.

Are there other instances of color selection that might benefit from being switched to use the color type input field?

randx commented:
@atomaka i think its only broadcast messages and labels. But labels already switched. THank you
